fuseimg/data/ops/image_loader.py

```diff
@@ -37,29 +37,39 @@ def reverse(self, sample_dict: dict, key_to_reverse: str, key_to_follow: str, op
 
 class OpLoadDicom(OpBase):
     '''
-    Loads a medical image, currently only nii is supported
+    Loads a medical image, currently only dicom is supported
     '''
     def __init__(self, dir_path: str , **kwargs):
+        '''
+        :param dir_path: the main folder path where the images located
+        '''
         super().__init__(**kwargs)
         self._dir_path = dir_path
 
-    def __call__(self, sample_dict: NDict, key_in:str, key_out: str, format:str="infer"):
+    def __call__(self, sample_dict: NDict, key_in:str, key_out: str, format:str="infer", metadata_keys: NDict=None, key_metadata_out: str="data.input.metadata"):
```

Comment:
add documentation of the new args
key_metadata_out default should be None, for a case that you don't want to read argments

Reply:
metadata_keys could be a simple dict (no need for NDict here)

Reply:
fixed